pkg/storage/tpr/storage_versioner.go

```diff
@@ -17,45 +17,92 @@ limitations under the License.
 package tpr
 
 import (
+	"errors"
 	"strconv"
 
+	"github.com/golang/glog"
 	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_5"
 	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/storage"
 )
 
 type storageVersioner struct {
+	codec        runtime.Codec
 	singularKind Kind
 	listKind     Kind
 	checkObject  func(runtime.Object) error
 	cl           clientset.Interface
+	defaultNS    string
 }
 
 // UpdateObject sets storage metadata into an API object. Returns an error if the object
 // cannot be updated correctly. May return nil if the requested object does not need metadata
 // from database.
 func (t *storageVersioner) UpdateObject(obj runtime.Object, resourceVersion uint64) error {
-	// if err := t.checkObject(obj); err != nil {
-	// 	return err
-	// }
-	// namespace, err := accessor.Namespace(obj)
-	// if err != nil {
-	// 	return err
-	// }
-	// accessor.SetResourceVersion(obj, fmt.Sprintf("%d", resourceVersion))
-	// unstruc, err := ToUnstructured(obj)
-	// if err != nil {
-	// 	return err
-	// }
-	// cl, err := GetResourceClient(t.cl, t.singularKind, namespace)
-	// if err != nil {
-	// 	return err
-	// }
-	// if _, err := cl.Update(unstruc); err != nil {
-	// 	return err
-	// }
+	if err := accessor.SetResourceVersion(obj, strconv.Itoa(int(resourceVersion))); err != nil {
+		glog.Errorf("setting resource version (%s)", err)
```

Comment:
no need - errors conform to `fmt.Stringer`, so `%s` will print out a good, human-readable message